kaolin/graphics/dib_renderer/rasterizer/rasterizer.py

```diff
@@ -0,0 +1,242 @@
+# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+
+from __future__ import print_function
+from __future__ import division
+
+import torch
+import torch.nn
+import torch.autograd
+from torch.autograd import Function
+
+import kaolin.graphics.dib_renderer.cuda.rasterizer as cuda_rasterizer
```

Comment:
This line makes my `import kaolin` fails, please use:
`from ..cuda import rasterizer as cuda_rasterizer`
or (but I would prefer relative import)
`from kaolin.graphics.dib_renderer.cuda import rasterizer as cuda_rasterizer`

Reply:
Done; Also fixed other relative imports